lib/dependencies/poetry.ts

```diff
@@ -33,6 +33,7 @@ export async function getPoetryDependencies(
       includeDevDeps
     );
     const plugin = await getMetaData(command, [], root, targetFile);
+    plugin.targetFile = manifestPath;
```

Comment:
More suitable place to set the `targetFile` will be inside the `getMetaData` function